PagerDuty-Services-Integration/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "pagerduty-services-integration",
+    "version": "0.1.0",
+    "description": "AWS custom resource provider named PagerDuty::Services::Integration.",
+    "private": true,
+    "main": "dist/handlers.js",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "build": "npx tsc",
+        "prepack": "npm run build",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "dependencies": {
+        "@amazon-web-services-cloudformation/cloudformation-cli-typescript-lib": "^1.0.3",
```

Comment:
Should this be 1.0.3 specifically (like https://github.com/aws-ia/cloudformation-pagerduty-resource-providers/pull/43/files#diff-3f88dee1706468c6e94c5df507156ebe4d88bfba40ca366a6bb85256452e36c2) or 1.0.3 and above

Reply:
done